.github/workflows/release.yml

```diff
@@ -19,6 +19,17 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.7
+      - name: Check Maven version
+        run: mvn -version
+      - name: Setup Maven
+        uses: s4u/setup-maven-action@v1.8.0
+        with:
+          checkout-fetch-depth: 0
+          java-version: 7
+          java-distribution: zulu
+          maven-version: 3.8.2
+      - name: Check Maven version again
+        run: mvn -version
```

Comment:
```yaml
- name: Set up Maven
  uses: stCarolas/setup-maven@v5
  with:
    maven-version: 3.8.2
- name: Check Maven version
  run: mvn -version
```
Let's try this, maybe it works.

Reply:
That *did* work. Thank you for the suggestion!